src/main/webapp/lectures/index.html

```diff
@@ -58,7 +58,16 @@ <h3 class="text-center">Add a Lecture</h3>
           <form name="lectureForm" action="/lecture" method="post">
             <input type="text" class="form-control" placeholder="Lecture Name" name="name-input" required />
             <br />
-            <input type="url" class="form-control" placeholder="Lecture YouTube URL" name="link-input" required />
+            <input
+              type="url"
+              class="form-control"
+              placeholder="Lecture YouTube URL"
+              name="link-input"
+              oninput="fetchAndDisplayTranscriptLanguages(this)"
+              required
+            />
+            <br />
```

Comment:
nit: I don't think there should be an empty space here.

Reply:
I noticed this is everywhere in the code. Is this normal?

Reply:
I believe so, the linter made this change.

Reply:
Ah ok, thanks